### PR TITLE
Add inference scripts for Ahmed body

### PR DESCRIPTION
**Pull request by ktangsali**
# Modulus Pull Request

## Description
Add inference notebook for Ahmed body

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Comment by ktangsali** [comment]
To be merged after: https://github.com/NVIDIA/modulus/pull/544

File: examples/cfd/ahmed_body_mgn/utils.py [hunk 1]
```diff
@@ -15,34 +15,96 @@
 # limitations under the License.
 
 import torch
+import numpy as np
+import pyvista as pv
+import shapely
 
 
-def compute_drag_coefficient(normals, area, coeff, p, s):
+def compute_frontal_area(mesh: pv.PolyData, direction: str = "x"):
```
Comment (Alexey-Kamenev) [inline examples/cfd/ahmed_body_mgn/utils.py:23]: Great - now we are computing the frontal area properly!!

**Review by Alexey-Kamenev — APPROVED** [review]

File: examples/cfd/ahmed_body_mgn/utils.py [hunk 2]
```diff
@@ -15,34 +15,96 @@
...
+                )
+            )
+            for tri in areas_proj.triangulate().regular_faces
+        ]
+    )
+
+    return merged.area
+
+
+def compute_drag_coefficient(
```
Comment (RishikeshRanade) [inline examples/cfd/ahmed_body_mgn/utils.py:73]: We discussed about adding a function for calculating lift. Another quantity of interest is the skin friction coefficient. It will be great if we can add that as it will also be useful for internal flow usecases. It is basically the local wall shear stress divided by the free stream kinetic energy and is calculated on each facet.

Comment (ktangsali) [reply examples/cfd/ahmed_body_mgn/utils.py]: Thanks @RishikeshRanade ! I modified the compute force calc utility to take arbitrary direction - now it can be used to compute drag or lift. Let me look into adding something for the skin friction

Comment (RishikeshRanade) [reply examples/cfd/ahmed_body_mgn/utils.py]: Yes, its basically c_f but normalized by the free-stream kinetic energy. It would be great if we can split those 2, it will make them reusable for other applications.

**Review by RishikeshRanade — APPROVED** [review]

File: examples/cfd/ahmed_body_mgn/utils.py (hunk 2 above)
Comment (ktangsali) [reply examples/cfd/ahmed_body_mgn/utils.py]: Thanks! modified the utility to return cp and cf separately. Resolving this thread now
